code/modules/events/anomaly_bluespace.dm

```diff
@@ -43,9 +43,12 @@
 				var/y_distance = TO.y - FROM.y
 				var/x_distance = TO.x - FROM.x
 				for(var/atom/movable/A in range(12, FROM )) // iterate thru list of mobs in the area
-					if(istype(A, /obj/item/radio/beacon)) continue // don't teleport beacons because that's just insanely stupid
-					if(A.anchored) continue
-
+					if(istype(A, /obj/item/radio/beacon)) // don't teleport beacons because that's just insanely stupid
+						continue 
+					if(istype(A, /mob/dead/observer)) // don't teleport ghosts.
```

Comment:
Might be better to use "if(isobserver(A))"

https://github.com/ParadiseSS13/Paradise/blob/bae5faae27c8589e158cc9279947b4aea1c953e2/code/__DEFINES/mobs.dm#L231